### ISSUE TITLE
Don't suggest disabling NTP

### ISSUE THREAD

**Issue by bknowles** [kristovatlas/osx-config-check#194]
Time synchronization is very important in computer security, and becoming more and more important as computers and code evolves.  So, why would you default to:

`systemsetup setusingnetworktime off`

Why?

If there is a legitimate security risk that needs to be addressed here, then let's do that.  We shouldn't be turning off NTP.

**Comment by kristovatlas** [comment]
I agree. NTP is one of those protocols that has had vulnerabilities in the past, but not so commonly that disabling likely outweighs the costs.

https://support.apple.com/en-us/HT204425
